server/server.go

```diff
@@ -41,9 +45,12 @@ func init() {
 	flag.DurationVar(&config.LifeWindow, "lifetime", 100000*100000*60, "Lifetime of each cache object.")
 	flag.IntVar(&config.HardMaxCacheSize, "max", 8192, "Maximum amount of data in the cache in MB.")
 	flag.IntVar(&config.MaxEntrySize, "maxShardEntrySize", 500, "The maximum size of each object stored in a shard. Used only in initial memory allocation.")
-	flag.IntVar(&port, "port", 9090, "The port to listen on.")
+	flag.IntVar(&port, "port", 443, "The port to listen on.")
```

Comment:
Can we use `defaultPort` here? I'm not a big fan of changing the default behaviour. We need to add readme entry about this change and short instruction how to run cache.

Reply:
Sure.